src/AM.Condo/Targets/Prepare/Node.targets

```diff
@@ -43,7 +43,8 @@
 
       <NpmInstall Condition=" '$(SKIP_NPM_INSTALL)' != '' ">false</NpmInstall>
       <NpmInstall Condition=" '$(NpmInstall)' != '' AND '$(NpmInstall.ToLower())' != 'true' ">false</NpmInstall>
-      <NpmInstall Condition=" '$(NpmInstall)' == '' ">$(NpmEnabled)</NpmInstall>
+      <NpmInstall Condition=" '$(NpmInstall)' == '' AND @(NodeInstallProjects->Count()) > 0 ">$(NpmEnabled)</NpmInstall>
+      <NpmInstall Condition=" '$(NpmInstall)' == '' ">false</NpmInstall>
```

Comment:
Why did this file need changes?

Reply:
it was an outstanding bug I rolled in -- npm install was being called regardless of whether or not a package.json file existed and since the new npm creates a project-lock.json file, it would add one to C# repositories.